staging/src/k8s.io/apiextensions-apiserver/Godeps/Godeps.json

```diff
@@ -1,2646 +0,0 @@
-{
```

Comment:
can you drop the Godeps.json changes from all the commits? these no longer exist

Reply:
yeah will do, sorry I thought `update-vendor.sh` cleaned those up automatically

Reply:
I ended up deleting all the Godeps file (stale updates cause this branch was created before go modules) but I noticed apiextensions-apiserver and kube-aggregator still had Godeps files in them in master. Are those special cases or did we just forgot?

Reply:
those are spurious, I'll open a PR to drop them